doc/examples.rst

```diff
@@ -8,17 +8,9 @@ The repository contains a number of simple examples for building applications
 with different frameworks:
 
 - `A console application <https://github.com/takluyver/pynsist/tree/master/examples/console>`_
-- `A tkinter application <https://github.com/takluyver/pynsist/tree/master/examples/tkinter>`_
-- `A PyQt4 application <https://github.com/takluyver/pynsist/tree/master/examples/pyqt4>`_
 - `A PyQt5 application <https://github.com/takluyver/pynsist/tree/master/examples/pyqt5>`_
-
   - `PyQt5 with QML <https://github.com/takluyver/pynsist/tree/master/examples/pyqt5_qml>`_
   - `PyQt5 with OpenCV <https://github.com/takluyver/pynsist/tree/master/examples/pyqt5_opencv>`_
-
```

Comment:
Sphinx likes blank lines around a sub-list like this. I think it still works without, but it produces a warning, and I try to keep the docs building without warnings, 'cause it's easy to get swamped in a million warnings.

Reply:
Ok! I have not used Sphinx before, so I was unaware of this. Thanks!